### PR TITLE
introduction.ipynb tutorial document: typo fixes, better explaination for "Time Travel" section

### PR DESCRIPTION
**Pull request by Robert-Jia00129**
Added more explanation regarding how state history and timestamp works.

### CODE REVIEW

File: examples/introduction.ipynb
Comment (hinthornw) [inline examples/introduction.ipynb:1]: Out of curiosity, why change the model to gpt-4o from haiku?

Comment (Robert-Jia00129) [reply examples/introduction.ipynb:1]: I ran out of credits....

Changed it back. My bad.

Comment (Robert-Jia00129) [reply examples/introduction.ipynb:1]: Also hi William I wanted to reach out to you regarding the script you wrote in LangChain that automatically generates API references. I want to change it to also incorporate API reference for `langgraph`, but encountered some issue. What's a good way to reach you?

Comment (hinthornw) [reply examples/introduction.ipynb:1]: You can write me at will@langchain.dev!

**Review by hinthornw — APPROVED** [review]

**Comment by vbarda** [comment]
Thank you for your contribution! Going to close since there are a lot of merge conflicts (too many changes since). If you'd still like to make these, feel free to open a fresh PR!

**Pull request closed without merging**